crates/sui-indexer/src/db.rs

```diff
@@ -128,13 +135,63 @@ pub fn get_pool_connection(pool: &ConnectionPool) -> Result<PoolConnection, Inde
     })
 }
 
+#[derive(QueryableByName)]
+pub struct StoredMigration {
+    #[diesel(sql_type = diesel::sql_types::Text)]
+    pub version: String,
+}
+
+pub async fn check_db_migration_consistency(pool: ConnectionPool) -> Result<(), IndexerError> {
```

Comment:
I saw this pr is targeting `main`, shall we base it on `idx-breaking-change-park` as we have some `migrations` changes there, which will be problematic to merge if this pr merges first.

Reply:
Could you elaborate what the problem is if we merge this to main today? Did we introduce migrations to main without a parking branch?